website/docs/reference/global-configs.md

```diff
@@ -103,6 +103,36 @@ config:
 
 </File>
 
+<VersionBlock firstVersion="1.1">
+
+### Cache database objects for selected resource
+
+:::caution Experimental config flag
+This should be considered an _unstable_ config flag because it is experimental and subject to change. We reserve the right to make breaking changes to this config flag.
+:::
+
+When true, caches database objects relevant to selected resource. Otherwise, will cache all database objects related to the project.
```

Comment:
```suggestion
When true, dbt caches database objects relevant to selected resource. Otherwise, dbt will cache all database objects related to the project. 

For example, if you wanted to improve speed and performance while focused on Salesforce models:
```
$ dbt build --select source:salesforce` 
$ dbt run --cache-selected-only` 
```
After selecting the source and using the `cache-selected-only` flag, you can start working with your salesforce models, and dbt will only cache those models instead of the entire project.

```